maps/away/away_site/orion_automated_station/orion_automated_station.dm

```diff
@@ -0,0 +1,253 @@
+	..()
+
+// -------------------------------- landmarks docks
+
+/obj/effect/shuttle_landmark/orion_automated_station
+	name = "orion_automated_station parent landmark"
+	base_area = /area/space
+	base_turf = /turf/space
+
+/obj/effect/shuttle_landmark/orion_automated_station/dock/west
+	name = "Dock, West"
+	landmark_tag = "nav_ox_auto_station_dock_west"
+	docking_controller = "airlock_ox_auto_station_dock_west"
+
+/obj/effect/shuttle_landmark/orion_automated_station/dock/east
+	name = "Dock, East"
+	landmark_tag = "nav_ox_auto_station_dock_east"
+	docking_controller = "airlock_ox_auto_station_dock_east"
+
+/obj/effect/shuttle_landmark/orion_automated_station/dock/north_1
+	name = "Dock, North 1"
+	landmark_tag = "nav_ox_auto_station_dock_north_1"
+	docking_controller = "airlock_ox_auto_station_dock_north_1"
+
+/obj/effect/shuttle_landmark/orion_automated_station/dock/north_2
+	name = "Dock, North 2"
+	landmark_tag = "nav_ox_auto_station_dock_north_2"
+	docking_controller = "airlock_ox_auto_station_dock_north_2"
+
+/obj/effect/shuttle_landmark/orion_automated_station/dock/south_1
+	name = "Dock, South 1"
+	landmark_tag = "nav_ox_auto_station_dock_south_1"
+	docking_controller = "airlock_ox_auto_station_dock_south_1"
+
+/obj/effect/shuttle_landmark/orion_automated_station/dock/south_2
+	name = "Dock, South 2"
+	landmark_tag = "nav_ox_auto_station_dock_south_2"
+	docking_controller = "airlock_ox_auto_station_dock_south_2"
+
+// -------------------------------- landmarks catwalks
+
+/obj/effect/shuttle_landmark/orion_automated_station/catwalk/north
+	name = "Catwalk, North"
+	landmark_tag = "nav_ox_auto_station_catwalk_north"
+
+/obj/effect/shuttle_landmark/orion_automated_station/catwalk/south
+	name = "Catwalk, South"
+	landmark_tag = "nav_ox_auto_station_catwalk_south"
+
+// -------------------------------- landmarks space
+
+/obj/effect/shuttle_landmark/orion_automated_station/space/north_east
+	name = "Space, North East"
+	landmark_tag = "nav_ox_auto_station_space_north_east"
+
+/obj/effect/shuttle_landmark/orion_automated_station/space/north_west
+	name = "Space, North West"
+	landmark_tag = "nav_ox_auto_station_space_north_west"
+
+/obj/effect/shuttle_landmark/orion_automated_station/space/south_east
+	name = "Space, South East"
+	landmark_tag = "nav_ox_auto_station_space_south_east"
+
+/obj/effect/shuttle_landmark/orion_automated_station/space/south_west
+	name = "Space, South West"
+	landmark_tag = "nav_ox_auto_station_space_south_west"
+
+// -------------------------------- areas
+
+/area/orion_automated_station
+	name = "orion_automated_station parent area"
+
+/area/orion_automated_station/center
+	name = "Central Hallway/Docks"
+	icon_state = "hallC"
+	area_blurb = "\
+	You hear a robotic voice coming from cheap ceiling-mounted speakers: \
+	'Welcome to the Orion Express Automated Station, where you may rest, exchange packages, and use any other of our facilities.' <br><br>\
```

Comment:
I wonder if this part could be made to use a different kind of text span, as it is actively speaking

Reply:
I don not see a nice way to do this, as macros can't really be used in strings